lib/web/desktop.go

```diff
@@ -31,9 +32,11 @@ import (
 	"github.com/gravitational/teleport/api/types"
 	"github.com/gravitational/teleport/lib/reversetunnel"
 	"github.com/gravitational/teleport/lib/srv/desktop"
+	"github.com/gravitational/teleport/lib/srv/desktop/tdp"
 	"github.com/gravitational/teleport/lib/utils"
 )
 
+// GET /webapi/sites/:site/desktops/:desktopName/connect?access_token=<bearer_token>&username=<username>&initial_width=<initialWidth>&initial_height=<initialHeight>
```

Comment:
this is minor but just for consistency reasons i would rename 

`handleDesktopAccessWebsocket -> to deskopAccessHanlde` :) so it's aligned with all other endpoints.

Reply:
`desktopConnectHandle` maybe?